packages/dapp/src/utils/selectProofFormat.ts

```diff
@@ -2,6 +2,8 @@ export const selectProofFormat = (method: string) => {
   switch (method) {
     case 'did:ethr':
     case 'did:pkh':
+    case 'did:jwk':
```

Comment:
This is probably ok for now, but maybe we need to think how to change this function if we want to support multiple proof formats for some methods. Example: `did:jwk` can be used for both `sd-jwt` and `jwt` and `did:ethr` can be used for both `sd-jwt` and `EthereumEip712Signature2021`